### PR TITLE
Follow/unfollow user

### PR DESCRIPTION
**Pull request by agrawalshanu708**
Add a follow/unfollow feature
image

### CODE REVIEW

File: src/comps/FollowUser/FollowUser.jsx [hunk 1]
```diff
@@ -7,11 +7,15 @@ import { getAllUser } from "../../redux/slice/user/userService";
 import { useDispatch, useSelector } from "react-redux";
 
 const FollowUser = () => {
+  const state = useSelector((store) => store.users)
+
 const dispatch = useDispatch();
   useEffect(() => {
+    console.log("running")
```
Comment (shashikantmaurya0007) [inline src/comps/FollowUser/FollowUser.jsx:14]: remove conslole

Comment (shashikantmaurya0007) [reply src/comps/FollowUser/FollowUser.jsx:14]: useeffect should be the last statement in logical section of component

**Review by shashikantmaurya0007 — APPROVED** [review]
i suggested few changes rest looks good

**Review by harshitpaliwal95 — APPROVED** [review]
Good file folder structure 
Looks good

**Pull request merged**